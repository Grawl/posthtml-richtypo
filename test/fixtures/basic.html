<!doctype html>
<p data-richtypo>There are 1000 "rules" to enrich your text</p>
