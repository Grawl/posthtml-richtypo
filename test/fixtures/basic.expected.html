<!doctype html>
<p data-richtypo>There are 1000 «rules» to&nbsp;enrich your&nbsp;text</p>
